cloudformation/media-atom-maker.json

```diff
@@ -345,17 +344,16 @@
                                         ]
                                     ]
                                 },
-                                "aws s3 cp 's3://atom-maker-conf/media-atom-maker.",
+                                "/opt/features/native-packager/install.sh -b atom-maker-dist -a media-atom-maker -t tgz\n",
+
+                                "mkdir /etc/gu \n",
+                                "aws s3 cp 's3://atom-maker-conf/",
                                 {
-                                    "Ref": "Stage"
+                                  "Ref": "Stage"
                                 },
-                                ".application.conf' ",
-                                "/home/media-service/media-atom-maker/conf/application.conf\n",
-                                "cat >> /home/media-service/media-atom-maker/conf/application.conf<<'EOF'\n",
-                                "aws.dynamo.tableName=", { "Ref": "AtomMakerTable" }, "\n",
```

Comment:
The only argument I can think of against removing these is that they are passed in as params - if the params change but the file in s3 doesn't then bad things happen. We should probably only worry about it if it becomes an issue, however...